frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetAccessManagement/ClusterSetAccessManagement.tsx

```diff
@@ -301,7 +301,13 @@ function AddUsersModal(props: {
     }
 
     return (
-        <AcmModal variant={ModalVariant.medium} title={t('access.add.title')} isOpen={props.isOpen} onClose={reset}>
+        <AcmModal
+            variant={ModalVariant.medium}
+            style={{ overflow: 'visible' }}
```

Comment:
@KevinFCormier In your view, should this change be applied to the ui-component instead?

Reply:
Last time we had an issue like this, I think we used `menuAppendTo="parent"` on the `AcmSelect`... would that work here?

Reply:
Thanks, appendTo() is the more elegant solution for sure.